File: dot-net-app/Ishod1/Model/Driver.cs
namespace Ishod1.Model
{
    class Driver
    {
        public int IDDriver { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string Telephone { get; set; }
        public int DrivingLicenceNumber { get; set; }

        public Driver(string firstName, string lastName, string telephone, int drivingLicenceNumber)
        {
            Firstname = firstName;
            Lastname = lastName;
            Telephone = telephone;
            DrivingLicenceNumber = drivingLicenceNumber;
        }

        public Driver(int idDriver, string firstName, string lastName, string telephone, int drivingLicenceNumber)
            : this(firstName, lastName, telephone, drivingLicenceNumber)
            => IDDriver = idDriver;

        public override string ToString() => $"{Firstname} {Lastname}";
    }
}

File: java-app/src/hr/algebra/model/Driver.java
package hr.algebra.model;

/**
 *
 * @author efurkev
 */
public class Driver {

    public int idDriver;
    public String firstname;
    public String lastname;
    public String telephone;
    public int drivingLicenceNumber;

    public Driver(String firstname, String lastname, String telephone, int drivingLicenceNumber) {
        this.firstname = firstname;
        this.lastname = lastname;
        this.telephone = telephone;
        this.drivingLicenceNumber = drivingLicenceNumber;
    }

    public Driver(int idDriver, String firstname, String lastname, String telephone, int drivingLicenceNumber) {
        this(firstname, lastname, telephone, drivingLicenceNumber);
        this.idDriver = idDriver;
    }

    public int getIdDriver() {
        return idDriver;
    }

    public void setIdDriver(int idDriver) {
        this.idDriver = idDriver;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getTelephone() {
        return telephone;
    }

    public void setTelephone(String telephone) {
        this.telephone = telephone;
    }

    public int getDrivingLicenceNumber() {
        return drivingLicenceNumber;
    }

    public void setDrivingLicenceNumber(int drivingLicenceNumber) {
        this.drivingLicenceNumber = drivingLicenceNumber;
    }

    @Override
    public String toString() {
        return "Driver{" + "idDriver=" + idDriver + ", firstname=" + firstname + ", lastname=" + lastname + '}';
    }
}
